packages/terra-abstract-modal/src/terra-dev-site/test/abstract-modal/AbstractModalFallbackFocus.test.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import AbstractModal from '../../../AbstractModal';
+import './AbstractModalTestStyles.module.scss';
+
+class AbstractModalCloseOnOutsideClick extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      isOpen: false,
+    };
+
+    this.focusTrapRef = React.createRef();
+    this.handleOpenModal = this.handleOpenModal.bind(this);
+    this.handleCloseModal = this.handleCloseModal.bind(this);
+  }
+
+  handleOpenModal() {
+    this.setState({ isOpen: true });
+  }
+
+  handleCloseModal() {
+    this.setState({ isOpen: false });
+  }
+
+  render() {
+    return (
+      <div>
+        <AbstractModal
+          ariaLabel="Modal disable close on outside click"
+          isOpen={this.state.isOpen}
+          closeOnOutsideClick
+          onRequestClose={this.handleCloseModal}
+          fallbackFocus={() => (this.focusTrapRef.current)}
```

Comment:
Would be valuable to add an test example for each prop type? String, function and node reference?

Reply:
Added the tests.